ethereum/.env.arbitrum.testnet

```diff
@@ -6,4 +6,13 @@ INIT_SIGNERS=["0x13947Bd48b18E53fdAeEe77F3473391aC727C638"]
 INIT_CHAIN_ID=23
 INIT_GOV_CHAIN_ID=0x1
 INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
-INIT_EVM_CHAIN_ID=421611
+INIT_EVM_CHAIN_ID=421613
+
+# Bridge Migrations
+BRIDGE_INIT_CHAIN_ID=23
+BRIDGE_INIT_GOV_CHAIN_ID=0x1
+BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+# Not sure if there should be a WETH, confirm this before mainnet deploy!
+BRIDGE_INIT_WETH=0x0000000000000000000000000000000000000000
```

Comment:
Agree that Portal probably shouldn't support sending ETH since it's native Ethereum ETH and so that would be a double-wrapped situation. Oddly enough though, there seems to be an Ethereum WETH bridged token on Arbitrum but no Arbitrum ETH (which is the native bridged Ethereum ETH) ERC20. 🤷